reference_entities/refresh_records.rst

```diff
@@ -0,0 +1,41 @@
+Refresh records completeness
+============================
+
+.. note::
+
+   Reference Entities feature is only available for the **Enterprise Edition**.
+
+Usecases
+--------
+
+Let's say we have some records having a value for each of those attributes:
+
+- 1 attribute of type Single Option
+- 1 attribute of type Multiple Option
+- 1 attribute of type Reference entity single link
+- 1 attribute of type Reference entity multiple links
+
+Whenever an option is removed from the Option (or Option collection) attribute, the records having the deleted option have to be refreshed in order to correctly compute the completeness.
+
+The same operation needs to happen if a record referenced by another record (through a Reference entity single link or Reference entity multiple links attribute) is deleted.
+All the records referencing the deleted record needs to be refreshed.
+
+.. note::
+
+   The record's data coming out of the API will always be accurate, yet filtering on the records' completeness may be de-synchronised.
```

Comment:
as the completeness on a single record edit page

Reply:
It "heavies" the sentence, I'm not sure I want to tell this. (+ I think it's only for the record grid, the record edit page is fine no?)